api/go.mod

```
@@ -3,7 +3,8 @@ module github.com/fluxcd/source-controller/api
 go 1.14
 
 require (
-	k8s.io/api v0.18.8
-	k8s.io/apimachinery v0.18.8
+	github.com/fluxcd/pkg/apis/meta v0.0.2
+	k8s.io/api v0.18.9
+	k8s.io/apimachinery v0.18.9
 	sigs.k8s.io/controller-runtime v0.6.2
```

```suggestion
	sigs.k8s.io/controller-runtime v0.6.3
```
to match the controller go.mod